CONTRIBUTING.md

```diff
@@ -11,12 +11,12 @@
 2. Run `make test` in the project root directory.
 
 ## Developing guidelines
-1. If CRDs in `deploy/crds` are updated, the same file names located in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
-2. If `role.yaml`, `role_binding.yaml`, or `operator.yaml` in `deploy` are updated, the ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
-3. If `README.md` is updated, ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
-4. Be sure to run `make update` or `make test` before you finish a commit.
-5. Run `operator-sdk add` to add a new API or controller, for more details, please refer to https://github.com/operator-framework/operator-sdk.
-6. Run `operator-sdk generate k8s` and `operator-sdk generate openapi` after you change something in `pkg/apis`.
+- If CRDs in `deploy/crds` are updated, the same file names located in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+- If `role.yaml`, `role_binding.yaml`, or `operator.yaml` in `deploy` are updated, the ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+- If `README.md` is updated, ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+- Be sure to run `make update` or `make test` before you finish a commit.
```

Comment:
I like this better. I would make the "Be sure to...." bullet to be first in the list.
Other than that I approve